extensions/amp-next-page/amp-next-page.md

```diff
@@ -64,36 +68,151 @@ The pages should be inlined using a JSON format.
 </amp-next-page>
 ```
 
-If loading the next document is successful, everything after the placement of `amp-next-page` is removed from the current document. Typical use would be to include `amp-next-page` directly after the unique content of a given page: for example, at the end of a news article or recipe, but before the footer or other content repeated across articles.
+If loading the next document is successful it will be appended to the end of
+the current document as a child of the `amp-next-page` component, moving any 
+content after it further down the page. Because of this the component should
+usually be placed directly after the unique content of a given page: for 
+example, at the end of a news article or recipe, but before the footer or 
+other content repeated across articles.
+
+{% call callout('Note', type='note') %} For performance reasons the 
+component will render a maximum of three documents (total) on screen at one 
+time. This limit may be changed or removed in the future.
+
+### Recommendation box
+
+If the user reaches the end of a page before the next has loaded (or if the 
+next page fails to load), a box will be displayed with links to the next three 
+pages. This box will also be displayed after the maximum number of articles 
+have been rendered.
+
+The appearance of these links can be customized by styling the following 
+classes:
+
+- `.amp-next-page-links` for the containing element
+- `.amp-next-page-link` for an individual link
+- `.amp-next-page-image` for the link image
+- `.amp-next-page-text` for the link text
+
+### Separator
+
+A separator is rendered between each loaded document. By default this is 
+rendered as a full-width hairline. It can be customised by styling the `
+.amp-next-page-default-separator` class.
+
+Alternatively, you can specify a custom separator containing arbitrary HTML 
+content as a child of the `amp-next-page` component by using the `separator` 
+attribute.
+```html
+<amp-next-page src="https://example.com/config.json">
+  <div separator>
+    <h1>Keep reading</h1>
+  </div>
+</amp-next-page>
+```
+
+### Element hiding
```

Comment:
I thought the plan was just to remove everything after <amp-next-page> for everything except the final document automatically. This other approach looks reasonable, but takes extra effort on the part of the developer. What's the reasoning for doing it this way instead?

Reply:
Full reasoning is in the doc that was emailed around, but main reasons are that doing it automatically is a) a bit too magical and b) requires that every page includes `<amp-next-page>`, which is more restrictive.

Reply:
Ah — that's right. Sorry I had forgotten that discussion!